### ISSUE TITLE
non latin characters in migration name

### ISSUE THREAD

**Issue by dvdobrovolskiy** [aspnet/EntityFramework#8483]
Accidentally named migration with non latin characters like 
```
PM> add-migration ControlledObjects_щквук
To undo this action, use Remove-Migration.
add-migration : Value does not fall within the expected range.
At line:1 char:1
+ add-migration ControlledObjects_щквук
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (:) [Add-Migration], ArgumentException
    + FullyQualifiedErrorId : System.ArgumentException,Add-Migration
 
```

Now I am stuck, cant Remove-Migration because of build error
```
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS2001	Source file 'C:\Users\dima\documents\visual studio 2017\Projects\TNPK\TNPK\Data\Migrations\20170516112547_ControlledObjects_骢�.cs' could not be found.	TNPK	C:\Users\dima\documents\visual studio 2017\Projects\TNPK\TNPK\CSC	1	Active

```

**Comment by bricelam** [comment]
What version are you using? This looks like #6722 which was fixed in the 1.0.0-msbuild3 pre-release of tools.

**Comment by dvdobrovolskiy** [comment]
1.1.2
It seems like different. File name of migration was generated with trash instead of non latin characters. Renaming file solved this issue
